src/main/java/com/google/sps/data/GriddedData.java

```diff
@@ -0,0 +1,45 @@
+package com.google.sps.data;
+
+/**
+ * This class represents the AQI data grid used by the client for visualisation.
+ * This grid is a linear approximation of the latitude and longitude, which may be
+ * inaccurate due to the curvature of the earth
+ */
+public class GriddedData {
+  public int resolution; // the width & height of each grid cell, stored in metres
+  public double[][] data;
+
+  // the coordinates of the top left corner of the grid
+  public double originLat;
```

Comment:
Good question, I suspect it was because I forgot there was a Coordinates class when I first wrote it, and forgot to go back and change it. It is now one origin